nerfstudio/models/instant_ngp.py

```diff
@@ -102,28 +104,33 @@ def populate_modules(self):
         """Set the fields and modules."""
         super().populate_modules()
 
-        self.field = TCNNInstantNGPField(
+        if self.config.disable_scene_contraction:
+            scene_contraction = None
+        else:
+            scene_contraction = SceneContraction(order=float("inf"))
```

Comment:
@liruilong940607 is scene contraction meant to be used with the new multi-level grid? Should TCNNNerfactoField just not take the expanded aabb as in https://github.com/KAIR-BAIR/nerfacc/blob/master/examples/train_ngp_nerf_occ.py#L130

Reply:
The field and multi-level occ grid are completely separated with each other. The new occ grid does not support contraction anymore but the field can still do contraction. I didn't play around with the field to compare it with/without contraction so I just kept it as is. 

Plz share if you find anything!